utils/format.ts

```diff
@@ -0,0 +1,36 @@
+export const strikeThrough = (text: string): string => {
+  return `<s>${text}</s>`
+}
+
+// Function to zero pad hour or minute strings
+export const addZeroPadding = (time: number): string => {
+  if (time < 10) return '0' + time.toString()
+  return time.toString()
+}
```

Comment:
Apparently we can achieve the same result by doing `String(time).padStart(2, '0')`